x-pack/plugins/threat_intelligence/public/modules/indicators/components/indicator_field/indicator_field.tsx

```diff
@@ -20,19 +21,21 @@ export interface IndicatorFieldProps {
    * The field to get the indicator's value for.
    */
   field: string;
-  /**
-   * An object to know what type the field is ('file', 'date', ...).
-   */
-  fieldTypesMap: { [id: string]: string };
 }
 
 /**
  * Takes an indicator object, a field and a field => type object to returns the correct value to display.
  * @returns If the type is a 'date', returns the {@link DateFormatter} component, else returns the value or {@link EMPTY_VALUE}.
  */
-export const IndicatorField: VFC<IndicatorFieldProps> = ({ indicator, field, fieldTypesMap }) => {
+export const IndicatorField: VFC<IndicatorFieldProps> = ({ indicator, field }) => {
```

Comment:
as we have `field_label` should we name this one `field_value`? I just realized that it took me some time to remember what `field` means in this context